Setup/Model/ProductInstaller.php

```diff
@@ -153,6 +166,70 @@ private function createProtectionPlanProduct(AttributeSetInterface $attributeSet
         }
     }
 
+    /**
+     * Adds customizable options to the protection plan product
+     *
+     * @param Product $product
+     * @return void
+     * @throws SetupException
+     */
+    private function addOptionsToProtectionPlanProduct(Product $product)
+    {
+        try {
+            $default_values = [
+                'type' => 'field',
+                'price_type' => 'fixed',
+                'price' => '0.00',
+                'sort_order' => 0,
+                'is_require' => 1,
+            ];
+            
+            $options = [
+                [
+                    'title'      => 'Associated Product',
+                ],
+                [
+                    'title'      => 'Plan Type',
+                ],
+                [
+                    'title'      => 'Plan ID',
+                ],
+                [
+                    'title'      => 'Term',
+                ],
+                [
+                    'title'      => 'List Price',
+                    'is_require' => 0,
+                ],
+                [
+                    'title'      => 'Order Offer Plan Id',
+                    'is_require' => 0,
+                ],
+            ];
+    
+            foreach ($options as $arrayOption) {
+                // if value exists in first and second array, the value from the first array will be used
+                $optionData = array_merge($arrayOption, $default_values);
+
+                $option = $this->catalogOptionFactory->create();
```

Comment:
@dustin-dg do you know if there is a difference between this and a customizable option?

cc: @scott-taylor-extend

Reply:
This is the customizable options!

Reply:
I will add a screenshot for verification. This is `product.options` on the product model.

Reply:
On a side note, is this being used for SP at all? If so these options should not be required right? @scott-taylor-extend